tf-modules/archive/api.tf

```diff
@@ -149,7 +149,7 @@ resource "aws_lambda_function" "private_api" {
   handler          = "index.handler"
   role             = aws_iam_role.lambda_api_gateway.arn
   runtime          = "nodejs16.x"
-  timeout          = lookup(var.lambda_timeouts, "PrivateApiLambda", 100)
+  timeout          = lookup(var.lambda_timeouts, "PrivateApiLambda", 900)
```

Comment:
What was the reason for this update?    We should be *very* careful about upping this timeout.